application/common/components/Emailer.php

```diff
@@ -190,16 +192,27 @@ public function getSiteStatus()
     {
         return $this->getEmailServiceClient()->getSiteStatus();
     }
-    
+
+    /**
+     * @param string $messageType
+     * @return null|string
+     */
     protected function getSubjectForMessage(string $messageType)
     {
         if ( ! empty($this->subjects[$messageType]) && strpos($this->subjects[$messageType], '%') !== false) {
             return sprintf($this->subjects[$messageType], $this->otherDataForEmails['idpDisplayName'] ?? '');
         }
         return $this->subjects[$messageType] ?? null;
     }
-    
-    protected function getViewForMessage(string $messageType, string $format)
+
+    /**
+     * Retrieve the view text for the given message type and format. The view is read from a
```

Comment:
This may just be a matter of clarification, but this method actually gets the path to the view file for the given message type and format. That path is later passed to a render() function, which goes and gets that view file and uses it in generating the content for the email.

It may just be me, but the way I understood your method description made it sound like it actually got the content of the message for the given message type an format.